Library/Homebrew/cmd/commands.rb

```diff
@@ -16,12 +16,14 @@ class CommandsCommand < Command
     options do
       command "commands"
       desc "Show a list of built-in and external commands."
-      option option: "quiet", desc: "list only the names of commands without the header." do
+      option option: "quiet", desc: "list only the names of commands without the header.", switch: "n" do
```

Comment:
Instead of having `option:` and `switch:` be different keyword parameters, it might sense for them to just be one with the switch-form coming first by convention. That would save some verbosity:

```ruby
option "f", "foo", desc: "whatever"
```

or possibly:

```ruby
option "f", "foo" do
  desc "whatever"
end
```